larky/src/main/java/com/verygood/security/larky/modules/vgs/cerebro/text/analyzer/TextAnalyzerModule.java

```diff
@@ -0,0 +1,141 @@
+package com.verygood.security.larky.modules.vgs.cerebro.text.analyzer;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Sets;
+import com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.analyzers.DefaultTextPIIAnalyzer;
+import com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.analyzers.NoopTextPIIAnalyzer;
+import com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.dto.TextPIIEntity;
+import com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.spi.TextPIIAnalyzer;
+import net.starlark.java.annot.Param;
+import net.starlark.java.annot.ParamType;
+import net.starlark.java.annot.StarlarkBuiltin;
+import net.starlark.java.annot.StarlarkMethod;
+import net.starlark.java.eval.EvalException;
+import net.starlark.java.eval.Starlark;
+import net.starlark.java.eval.StarlarkFloat;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.ServiceLoader;
+import java.util.Set;
+
+@StarlarkBuiltin(
+    name = "TextPIIAnalyzer",
+    category = "BUILTIN",
+    doc = "Overridable Text PII Analysis API in Larky")
+public class TextAnalyzerModule implements TextPIIAnalyzer {
+
+  public static final TextAnalyzerModule INSTANCE = new TextAnalyzerModule();
+  public static final String ENABLE_INMEMORY_PROPERTY = "larky.modules.vgs.cerebro.piiAnalyzer.text.enableInMemory";
```

Comment:
This should match the namespace. Do we want to call it piiAnalzyer? Mind your camelCase as well. Is that what we normally use?

Reply:
Updated. We do for the leaf, like: https://github.com/verygoodsecurity/starlarky/blob/master/larky/src/main/java/com/verygood/security/larky/modules/VaultModule.java#L27